### PR TITLE
Improvement/process labels

### PR DESCRIPTION
**Pull request by mikibonacci**
This fixes https://github.com/aiidalab/aiidalab-qe/issues/811.

*Still missing*, capitalize PDOS everywhere in the GUI. Coming soon (but not really relevant for the PR review)

### CODE REVIEW

File: src/aiidalab_qe/app/submission/__init__.py [hunk 1]
```diff
@@ -369,17 +369,26 @@ def _update_process_label(self) -> dict:
...
+            protocol_and_magnetic_info += ", magnetic"
+        # properties_info
         if not properties:
             properties_info = ""
         else:
-            properties_info = f", properties on {', '.join(properties)}"
+            properties_info = f"+ {', '.join(properties)}"
 
-        label = f"{formula} {relax_info} {properties_info}"
+        label = f"{formula} ({relax_info}, {protocol_and_magnetic_info}) {properties_info}".strip()
```
Comment (AndresOrtegaGuerrero) [inline src/aiidalab_qe/app/submission/__init__.py:391]: Would it be nice to display the label users are given to the structure?

**Comment by AndresOrtegaGuerrero** [comment]
what a process label like this 

Si2 [Relax: Atoms/Cell, Moderate] ➔ Bands, Pdos, Vibro

I would prefer actually to use the label of the structure like 

7GNR [Relax: Atoms/Cell, Moderate] ➔ Bands, Pdos, Vibro

What do you think ?

File: src/aiidalab_qe/app/submission/__init__.py [hunk 2]
```diff
@@ -369,17 +369,26 @@ def _update_process_label(self) -> dict:
...
+        # relax_info
         relax_type = workchain_data.get("relax_type", "none")
-        if relax_type != "none":
-            relax_info = "structure is relaxed"
+        if relax_type == "none":
+            relax_info = "not relaxed"
         else:
-            relax_info = "structure is not relaxed"
+            relax_info = (
+                "relax: atoms+cell" if "cell" in relax_type else "relax: atoms only"
```
[the remark this answers is not in the document]
Comment (mikibonacci) [reply src/aiidalab_qe/app/submission/__init__.py:378]: I would keep lowercase everything (so also the *properties* do not need to be uppercased) and keep explicit the *protocol* word, as who is not expert in AiiDA may not immediately understand that we are indicating the protocol (in particular in the `moderate` case, I would say). 

I like the new format of your [comment](https://github.com/aiidalab/aiidalab-qe/pull/836#issuecomment-2389627465) and the new relax mapping, but I would keep `atoms+cell`, as `atoms/cell` may sound misleading (not immediately clear that is a `vc-relax`). Or maybe `atoms&cell`? But I don't have a strong preference.

What do you think, @AndresOrtegaGuerrero ? Am I too paranoic? 😃

[the remark this answers is not in the document]
Comment (AndresOrtegaGuerrero) [reply src/aiidalab_qe/app/submission/__init__.py:378]: Yeah the lowercase is not a problem. And I see your point of the protocol. Yes you can keep the "+" does the same effect. No you aren't paranoic

**Comment by AndresOrtegaGuerrero** [comment]
@mikibonacci 
image
If you noticed the Label for the job, the structure doesnt use the label assigned to the structure 

image

It could be nice that for the label for the job, it already uses the label that the user is giving to the structure

**Comment by mikibonacci** [comment]
Hi @AndresOrtegaGuerrero , I added the structure label as default label (then, if empty label is provided, we still use the formula). I also adapted to the format you suggested

File: tests/test_submit_qe_workchain.py [hunk 3]
```diff
@@ -25,6 +25,21 @@ def test_create_builder_default(
...
+):
+    """ "Test the creation of the correct process label"""
+
+    app = submit_app_generator(properties=["bands", "pdos"])
+    submit_step = app.submit_step
+    submit_step._update_process_label()
+    assert (
+        submit_step.process_label.value
+        == "Si2 [relax: atoms+cell, moderate protocol] → bands, pdos"
+    )
```
Comment (AndresOrtegaGuerrero) [inline tests/test_submit_qe_workchain.py:40]: Could we add an extra test , where the structure has a label ?

**Comment by mikibonacci** [comment]
Hi @AndresOrtegaGuerrero , I added (in the same test) the check if we provide a label != formula and an empty label (we fallback into the formula)

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM, thank you Miki

**Comment by mikibonacci** [comment]
I will just see if I can capitalize all the PDOS in the app, then I will merge

**Pull request merged**